vendor/assets/stylesheets/jstree.scss

```diff
@@ -403,7 +403,7 @@
 }
 .jstree-default .jstree-node,
 .jstree-default .jstree-icon {
-  background-image: url("32px.png");
+  background-image: url(image-path("32px.png"));
```

Comment:
since this is editing a `vendor/assets` can it be moved to `app/assets` ?

Reply:
Hmm, maybe. I didn't really like the solution of editing the CSS, but couldn't find any other way around it.

I got the idea from https://makandracards.com/makandra/29567-managing-vendor-libraries-with-the-rails-asset-pipeline#section-fixing-font-paths

I did wonder if there's a way to automatically do this as part of the asset-pipeline compilation, or how it works for the old rails-assets gems

Reply:
actually, there are plenty of other cases where files here have been edited